[debug] [2023-01-30T16:57:24.982Z] ----------------------------------------------------------------------
[debug] [2023-01-30T16:57:24.985Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\Usuario\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy
[debug] [2023-01-30T16:57:24.987Z] CLI Version:   11.20.0
[debug] [2023-01-30T16:57:24.987Z] Platform:      win32
[debug] [2023-01-30T16:57:24.987Z] Node Version:  v16.17.1
[debug] [2023-01-30T16:57:24.989Z] Time:          Mon Jan 30 2023 13:57:24 GMT-0300 (hora estÃ¡ndar de Uruguay)
[debug] [2023-01-30T16:57:24.989Z] ----------------------------------------------------------------------
[debug] 
[debug] [2023-01-30T16:57:25.170Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects/undefined [none]
[debug] [2023-01-30T16:57:26.194Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects/undefined 403
[debug] [2023-01-30T16:57:26.195Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects/undefined {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2023-01-30T16:57:26.197Z] HTTP Error: 403, The caller does not have permission
[debug] [2023-01-30T16:57:26.314Z] FirebaseError: HTTP Error: 403, The caller does not have permission
    at responseToError (C:\Users\Usuario\AppData\Roaming\npm\node_modules\firebase-tools\lib\responseToError.js:49:12)
    at RetryOperation._fn (C:\Users\Usuario\AppData\Roaming\npm\node_modules\firebase-tools\lib\apiv2.js:288:77)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
[error] 
[error] Error: Failed to get Firebase project undefined. Please make sure the project exists and your account has permission to access it.
